api/app/fire_behaviour/prediction.py

```diff
@@ -453,18 +454,38 @@ def calculate_fire_behaviour_prediction_using_c7b(latitude: float,
                                      percentage_dead_balsam_fir=None,
                                      ros=ros, cfb=cfb, cfl=cfl, sfc=sfc)
 
+    lb_ratio = cffdrs.length_to_breadth_ratio(FuelTypeEnum.O1A, wind_speed)
+    wsv = cffdrs.calculate_wind_speed(FuelTypeEnum.O1A,
+                                      ffmc=ffmc,
+                                      bui=bui,
+                                      ws=wind_speed,
+                                      fmc=fmc,
+                                      sfc=sfc,
+                                      pc=None,
+                                      cc=cc,
+                                      pdf=None,
+                                      cbh=cbh,
+                                      isi=isi)
+    bros = cffdrs.back_rate_of_spread(FuelTypeEnum.O1A,
+                                      ffmc=ffmc,
+                                      bui=bui,
+                                      wsv=wsv,
+                                      fmc=fmc, sfc=sfc,
+                                      pc=None,
+                                      cc=cc,
+                                      pdf=None,
+                                      cbh=cbh)
+    sixty_minute_fire_size = get_fire_size(FuelTypeEnum.O1A, ros, bros, 60, cfb, lb_ratio)
```

Comment:
why's O1A being used for these C7B calcs?

Reply:
Neal + Dana said that the fire size characteristics of C7B were similar to a grass fuel type, since the growth would be occurring on the ground. It's up to Dana+Neal to validate if this is correct.